drivers/tests/TESTS/mbed_drivers/lp_ticker/main.cpp

```diff
@@ -25,11 +25,14 @@
 #else
 
 using utest::v1::Case;
+using std::chrono::milliseconds;
+using std::chrono::microseconds;
 
 static const int test_timeout = 10;
 
 #define TICKER_COUNT 16
-#define MULTI_TICKER_TIME_MS 100
+#define MULTI_TICKER_TIME_MS 100ms
+#define MULTI_TICKER_TIME_US microseconds{MULTI_TICKER_TIME_MS}
```

Comment:
This extra define really shouldn't be needed. You can just pass `100ms` to the APIs that take microseconds.